tests/components/aemet/test_sensor.py

```diff
@@ -93,7 +93,7 @@ async def test_aemet_weather_create_sensors(
     assert state.state == "1004.4"
 
     state = hass.states.get("sensor.aemet_rain")
-    assert state.state == "1.8"
+    assert state.state == "7.0"
```

Comment:
What's the reason the state changed? Please point me to the code, which is responsible for this state change

Reply:
> What's the reason the state changed? Please point me to the code, which is responsible for this state change

In the following commit I introduced a change with respect to the current Home Assistant coordinator code, which gets the value reported by the weather station (7.0) and only fallbacks to the forecast value (1.8) if it's None:
https://github.com/Noltari/AEMET-OpenData/commit/280d9976e53d5bf487009878fe1b7c70a98cea06

---

The precipitation value from the hourly forecast:
https://github.com/home-assistant/core/blob/e157206eeba713e498ad6501c9fcc9601551804f/tests/components/aemet/fixtures/town-28065-forecast-hourly-data.json#L125-L128

The precipitation value from the weather station:
https://github.com/home-assistant/core/blob/e157206eeba713e498ad6501c9fcc9601551804f/tests/components/aemet/fixtures/station-3195-data.json#L447

Reply:
Alright, thanks for the clarification 👍 

../Frenck